src/Microsoft.IdentityModel.Tokens/opensource/json/SerializationBinder.cs

```diff
@@ -1,5 +1,5 @@
 ﻿
-#if (DOTNET || PORTABLE40 || PORTABLE || NET_CORE)
+#if (DOTNET || PORTABLE40 || PORTABLE)
```

Comment:
>PORTABLE40 || PORTABLE [](http://example.com/codeflow?start=15&length=22)

do we need these?

Reply:
We use NET_CORE in our test code, i don't think it is related to the product code, but i will add it back.
PORTABLE40 and PORTABLE are not defined in our builds, and are in many locations in the newtonsoft source.